reference-implementation/run-web-platform-tests.js

```diff
@@ -19,6 +19,9 @@ wptRunner(testsPath, setup)
   });
 
 function setup(window) {
+  // Necessary so that we can send test-realm promises to the jsdom-realm implementation without causing assimilation.
+  window.Promise = Promise;
```

Comment:
I lack knowledge to understand this. Should I just learn what wpt-runner is doing more?